monai/networks/nets/dynunet.py

```diff
@@ -115,6 +116,7 @@ def __init__(
         kernel_size: Sequence[Union[Sequence[int], int]],
         strides: Sequence[Union[Sequence[int], int]],
         upsample_kernel_size: Sequence[Union[Sequence[int], int]],
+        dropout: float = 0.0,
```

Comment:
I think in the new version we can use the latest API `dropout: Optional[Union[Tuple, str, float]] = None,`

Reply:
Agree! Just changed it